### PR TITLE
2020 User Survey

### PR DESCRIPTION
**Pull request by TomAugspurger**
Results are pretty similar to last year. I think the most interesting bits are that

* Our documentation / examples may be doing a better job than last year (relatively fewer people are using the resource tracker for help)
* Performance is relatively more important for more API groups than last year (people are using Dask for larger workloads?)

Note: I had to set the date to 2020-08-21.md to get this to build. We'll publish it sometime next week hopefully, so the date will need to move.

### CODE REVIEW

**Comment by jacobtomlinson** [comment]
@TomAugspurger when you say you had to set the date in the past to get it to build do you mean for testing it locally?

File: _posts/2020-08-21-survey.md [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+
+## What other systems do you use?
+
+SSH continues to be the most popular "cluster resource mananger". This was the big surprise last year, so we put in some work to make it nicer. Aside from that, not much has changed.
+
+<img src="/images/2020_survey/2020_25_0.png">
+
+And Dask users are about as happy with its stability as last year.
+
+<img src="/images/2020_survey/2020_25_0.png">
```
Comment (jacobtomlinson) [inline _posts/2020-08-21-survey.md]: I think this is the wrong graph

**Review by jacobtomlinson — COMMENTED** [review]
Thanks for putting this together @TomAugspurger. Looks great!

**Comment by TomAugspurger** [comment]
> @TomAugspurger when you say you had to set the date in the past to get it to build do you mean for testing it locally?

Yep. I've moved it to Tuesday in the hope that it works when we build it on CI.

**Comment by jacobtomlinson** [comment]
You should be able to run `jekyll serve --future` locally to make future posts show up while developing.

**Comment by TomAugspurger** [comment]
Moved the date to today. Merging.

**Pull request merged**